modules/openshift-developer-cli-mirroring-a-supported-builder-image.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// cli_reference/openshift_developer_cli/using_odo_in_a_restricted_environment/creating-and-deploying-a-component-to-the-disconnected-cluster
+
+
+
+.Procedure
```

Comment:
The procedure should come after the heading. But strangely I see the preview reflecting it correctly. 
Also, the [id="<module title>_{context}"] is missing.